modules/navier_stokes/test/tests/finite_volume/cns/pressure_outlet/tests

```diff
@@ -0,0 +1,11 @@
+[Tests]
+  issues = '#16758'
+  design = 'CNSFVHLLCBase.md'
+  [./fv_specified_pressure_out]
+    type = Exodiff
+    input =  subsonic_nozzle_fixed_inflow_hllc.i
+    exodiff = subsonic_nozzle_fixed_inflow_hllc_out.e
+    abs_zero = 1e-9
+    requirement = "The system shall be able to model subsonic nozzle flow using an HLLC discretization with a specified outlet pressure."
+  [../]
```

Comment:
I actually think the problem is probably inadequate BC specification. subsonic inlet needs dim + 1 specified properties, so 3 in this 2D case, and there are only 2. I know that @snschune has talked for a while about something wrong with specified pressure. Well we fixed one thing ... the elem information at the boundary should use the interior pressure as opposed to the boundary pressure function. But this may be a separate problem (e.g. the incorrect number of bcs)

Reply:
> this test does not converge beyond timestep 16 (stopped at 10 in this test).

Are you using a recent checkout of this branch that has the elem correction (removing the pressure function from the elem computations in the specified pressure BC)? I am able to run out to `num_steps = 20` without issue. The pressure profile with the current input does indeed look a bit weird

<img width="603" alt="Screen Shot 2021-06-23 at 4 53 01 PM" src="https://user-images.githubusercontent.com/10248304/123182132-a5884280-d443-11eb-8a66-c9ecb6c9eb5b.png">

and I do see the weird local maxima with `Ma_layered` 

<img width="692" alt="Screen Shot 2021-06-23 at 4 54 50 PM" src="https://user-images.githubusercontent.com/10248304/123182185-c05ab700-d443-11eb-8efe-d5516bcf5153.png">

but at least it runs

Reply:
Ah sorry I think I had bumped up the timesteps too. I was using the pre-review version of the cnfsv too. 
So with the current version it fails at like 6.5s with a 0.05s timestep.
With 0.01s timestep, I was able to reach to reach about the same time. 

The solution is hardly stabilized at this point, the specified pressure (9e4) is hardly obeyed from the beginning, as the top pictures show.
By the time it fails:
![image](https://user-images.githubusercontent.com/25037151/123197626-0413e780-d469-11eb-8237-2993d033e58e.png)

Reply:
Ok yea I think I'm going to see about how to fix the bad number of inlet bcs

Reply:
Ok yea I'm able to run to steady-state with sufficient BCs. And layered Mach issue was because of more layers than cells in the x-direction